src/OpenTK.Windowing.Common/EventingModels/MouseMoveEventArgs.cs

```diff
@@ -0,0 +1,64 @@
+//
+// MouseMoveEventArgs.cs
+//
+// Copyright (C) 2018 OpenTK
+//
+// This software may be modified and distributed under the terms
+// of the MIT license. See the LICENSE file for details.
+//
+
+namespace OpenToolkit.Windowing.EventingModels
+{
+    /// <summary>
+    /// Defines the event data for <see cref="MouseDevice.Move" /> events.
+    /// </summary>
+    /// <remarks>
+    ///  <para>
+    /// Do not cache instances of this type outside their event handler.
+    /// If necessary, you can clone an instance using the
+    ///  <see cref="MouseMoveEventArgs(MouseMoveEventArgs)" /> constructor.
+    ///  </para>
+    /// </remarks>
+    public class MouseMoveEventArgs : MouseEventArgs
+    {
+        /// <summary>
+        /// Constructs a new <see cref="MouseMoveEventArgs" /> instance.
+        /// </summary>
+        public MouseMoveEventArgs()
+        {
+        }
+
+        /// <summary>
+        /// Constructs a new <see cref="MouseMoveEventArgs" /> instance.
+        /// </summary>
+        /// <param name="x">The X position.</param>
+        /// <param name="y">The Y position.</param>
+        /// <param name="xDelta">The change in X position produced by this event.</param>
+        /// <param name="yDelta">The change in Y position produced by this event.</param>
+        public MouseMoveEventArgs(int x, int y, int xDelta, int yDelta)
```

Comment:
"MouseMoveEventArgs.cs(38, 53): [SA1305] parameter 'xDelta' should not use Hungarian notation"
Styleguide does not want to have the name like that. With this I would like to propose to rename it(and the properties as well). do DeltaX, it sounds a lot more natural to me. Afaik it is also the way how it would be spelled out(at least where I come from it is)...

Reply:
I agree, will change it.

Reply:
Fixed.